src/base_types.ts

```diff
@@ -6,6 +6,7 @@ export interface ContextParams {
   API_SECRET_KEY: string;
   SCOPES: string[] | AuthScopes;
   HOST_NAME: string;
+  PROTOCOL?: 'http:' | 'https:';
```

Comment:
```suggestion
  PROTOCOL?: string;
```
You should replace the strings with the string type